packages/opentelemetry-exporter-prometheus/src/PrometheusExporter.ts

```diff
@@ -56,8 +58,14 @@ export class PrometheusExporter implements MetricExporter {
     this._logger = config.logger || new NoopLogger();
     this._port = config.port || PrometheusExporter.DEFAULT_OPTIONS.port;
     this._prefix = config.prefix || PrometheusExporter.DEFAULT_OPTIONS.prefix;
+    this._appendTimestamp =
+      config.appendTimestamp ??
```

Comment:
please use `typeof config.appendTimestamp === 'boolean'` otherwise numbers strings values etc. will be treated as valid

Reply:
Thanks, @obecny . You mean something like this?
```
    this._appendTimestamp =
      typeof config.appendTimestamp === 'boolean' ?
      config.appendTimestamp :
      PrometheusExporter.DEFAULT_OPTIONS.appendTimestamp;
```

Reply:
yes exactly